CMakeLists.txt

```diff
@@ -913,7 +913,11 @@ if(BUILD_LIBRETRO)
 	add_library(${BINARY_NAME}_libretro SHARED ${CORE_SRC} ${RETRO_SRC})
 	set_target_properties(${BINARY_NAME}_libretro PROPERTIES PREFIX "" COMPILE_DEFINITIONS "COLOR_16_BIT;COLOR_5_6_5;DISABLE_THREADING;${OS_DEFINES};${FUNCTION_DEFINES};MINIMAL_CORE=2")
 	target_link_libraries(${BINARY_NAME}_libretro ${OS_LIB})
-	install(TARGETS ${BINARY_NAME}_libretro LIBRARY DESTINATION ${LIBRETRO_LIBDIR} COMPONENT ${BINARY_NAME}_libretro NAMELINK_SKIP)
+	if(MSVC)
```

Comment:
Why? I didn't think `NAMELINK_SKIP` did anything on MSVC anyway.

Reply:
```
cmake -G "Visual Studio 15 2017"

install(TARGETS ${BINARY_NAME}_libretro LIBRARY DESTINATION ${LIBRETRO_LIBDIR} COMPONENT ${BINARY_NAME}_libretro NAMELINK_SKIP)
==>
CMake Error at CMakeLists.txt:916 (install):
  install Library TARGETS given no DESTINATION!


install(TARGETS ${BINARY_NAME}_libretro RUNTIME DESTINATION ${LIBRETRO_LIBDIR} COMPONENT ${BINARY_NAME}_libretro NAMELINK_SKIP)
==>
CMake Error at CMakeLists.txt:916 (install):
  install TARGETS given NAMELINK_SKIP option not in LIBRARY group.  The
  NAMELINK_SKIP option may be specified only following LIBRARY.


install(TARGETS ${BINARY_NAME}_libretro RUNTIME DESTINATION ${LIBRETRO_LIBDIR} COMPONENT ${BINARY_NAME}_libretro)
==>
CMake okay
```

Reply:
Are you sure LIBRETRO_LIBDIR is set?

Reply:
Tried lots of different things but nothing else worked so far. Same missing DESTINATION error, even if I hardcode the path or tried other valid ones.


I've read these comments about cmake and windows
https://github.com/texane/stlink/issues/487#issuecomment-250801216
https://stackoverflow.com/questions/14990343/cmake-error-targets-given-no-library-destination-for-shared-library-target/37729574